tests/basis_test.py

```diff
@@ -74,9 +45,9 @@ def test_cheb_call_random_points_multi_input():
     f0 = ChebyshevFirstKind(0)
```

Comment:
These sorts of variations of values should be done using a parametrization rather than repeating the code 3 times.